src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -457,11 +496,24 @@ private void release()
             }
         }
 
+        private long challengeAfter()
+        {
+            return this.expiresAt - this.challengeResponseTimeout;
+        }
+
+        private boolean withinChallengeBuffer(
+            long currentTimeMillis,
+            long challengeAfter)
+        {
+            return currentTimeMillis >= challengeAfter && currentTimeMillis < expiresAt;
```

Comment:
`challengeAfter` is `challengeAfter()`, right? So no need to pass it in to the method.